src/components/overlay/FullscreenLoading.tsx

```diff
@@ -8,11 +8,11 @@ import {
   CircularProgress,
 } from "@chakra-ui/react"
 
-export interface Props extends FlexProps {
+export interface FullscreenLoadingProps extends FlexProps {
```

Comment:
Good catch, my bad! Might need a custom linter rule for ensuring nothing named `Props` is exported from a module

Reply:
Good idea. Otherwise, we're only alerted when there's a double up